sha/methods/guest/Cargo.toml

```diff
@@ -6,7 +6,13 @@ edition = "2021"
 [workspace]
 
 [build-dependencies]
-risc0-build = "0.11"
+risc0-build = "1.0.0-rc.2"
 
 [dependencies]
-risc0-zkvm-guest = "0.11"
+risc0-zkvm = { version = "1.0.0-rc.2", features = ["std"], default-features = false }
+
+[patch.crates-io]
+# TODO(victor): Remove these patch definitions when d02f606a is included in a release.
+risc0-zkp = { git = "https://github.com/risc0/risc0", rev = "d02f606ab7d59ad45dc75314eb3e0eef5c9a26b7"}
+risc0-zkvm = { git = "https://github.com/risc0/risc0", rev = "d02f606ab7d59ad45dc75314eb3e0eef5c9a26b7"}
+risc0-build = { git = "https://github.com/risc0/risc0", rev = "d02f606ab7d59ad45dc75314eb3e0eef5c9a26b7"}
```

Comment:
So I'm thinking we can use git references directly instead of doing patches, at least until a new release is out.